packages/web/src/components/premium-content-purchase-modal/components/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { PayExtraAmountPresetValues } from './types'
+
+export const CUSTOM_AMOUNT = 'customAmount'
+export const AMOUNT_PRESET = 'amountPreset'
+
+export const payExtraAmountPresetValues: PayExtraAmountPresetValues = [
```

Comment:
also nit: if we used a partial (or whole?) `Record` here (or made `NONE` index 0), we could have `NONE` map to 0?

Reply:
Alright I decided to convert the enum to be strings and index an object based on those strings. The shape of the preset amounts object that the component accepts now just requires an object with the `low`, `medium`, and `high` keys in it. This feels more readable than numbers (definitely during debugging or when reading through analytics events), and doesn't rely on ordering of the enum keys to index into an array properly.